code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -692,6 +701,12 @@ pub mod pallet {
 						.try_collect()
 						.expect("No items were added; qed;");
 
+					let share_asset_id = Self::register_protocol_asset(
+						ShareAssetNonce::<T>::increment().expect("Does not exceed `u64::MAX`"),
+					)?;
+					// NOTE: Collection ID can safely be set to the ID of the share asset
```

Comment:
Need further explanation here as I guess what you mean is these assets are name spaced because of fnft being a NFT asset not tracked by asset registry?

Reply:
yes, from my conversation with Connor yesterday

Reply:
Exactly - Connor and I confirmed with dzmitry that this is ok for xcm as well

Reply:
That's correct, I can clarify the comment.